mod-finance/schemas/fy_finance_data.json

```diff
@@ -107,6 +111,23 @@
     "groupCode": {
       "description": "The code for the group",
       "type": "string"
+    },
+    "transactionDescription": {
+      "description": "A description of this transaction",
+      "type": "string"
+    },
+    "transactionTag": {
+      "type": "object",
+      "description": "Arbitrary tags associated with this transaction",
+      "$ref": "../../../raml-util/schemas/tags.schema"
+    },
+    "updateType": {
```

Comment:
It should be at higher level in collection object, because we can have single updateType for every object in the array